android/assets/jsons/translations/French.properties

```diff
@@ -88,7 +88,7 @@ Requires a [buildingName] in this city = Nécessite un(e) [buildingName] dans ce
 Cannot be built with [buildingName] = Ne peut pas être construit avec [buildingName]
 Consumes [amount] [resource] = Consomme [amount] [resource]
  # Requires translation!
```

Comment:
this comment should be removed since translation below are done

Reply:
Comments are automatically removed when a new version is released, no need to do it manually.